README.md

```diff
@@ -63,7 +63,7 @@ These projects are setup to work with the VS Code Prettier ESLint extension. Use
 
 ### Install Dependencies
 
-1. In your project, install `eslint@6.8.0`, `prettier@1.9.1`, & `prettier-eslint:^10.1.0` as `devDependencies`
+1. In your project, install `eslint@6.8.0`, `prettier@1.9.1`, & `prettier-eslint@^10.1.0` as `devDependencies`
```

Comment:
Actually `prettier-eslint` is no longer required. It is bundled with the extension.